src/layout/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './layout.css';
+import { UsersBar } from '../users_bar/users_bar';
+import MockUsers from '../mock/users.json';
+import { UserType } from '../user/user';
+
+type Props = {};
+type State = {
+  users: Array<UserType>;
+};
+
+export class Layout extends React.Component<Props, State> {
```

Comment:
Rewrite to functional component with react [React hooks](https://ru.reactjs.org/docs/hooks-reference.html)

Reply:
component deleted